specification/frontdoor/resource-manager/Microsoft.Network/stable/2019-11-01/networkexperiment.json

```diff
@@ -1110,16 +1112,16 @@
         }
       ]
     },
-    "ResourceState": {
+    "NetworkExperimentationResourceState": {
       "description": "Defines the server side resource status",
       "readOnly": true,
       "enum": [
-        "creating",
-        "enabling",
-        "enabled",
-        "disabling",
-        "disabled",
-        "deleting"
+        "Creating",
```

Comment:
again, breaking change

Reply:
discussed with service team, they will make sure RP is in sync (networkexperiment was not release yet)

Reply:
yes. it's not breaking-- we will make sure to have these changes reflected in our service.